Tests/UI/New UI/BillingForm/ViewModel/PaymentViewModelTests.swift

```diff
@@ -0,0 +1,89 @@
+import XCTest
+import Checkout
+@testable import Frames
+
+class PaymentViewModelTests: XCTestCase {
+
+    var viewModel: DefaultPaymentViewModel!
+
+    override func setUp() {
+        super.setUp()
+        UIFont.loadAllCheckoutFonts
+    }
+
+    func testUpdateExpiryDateView() {
+        viewModel = DefaultPaymentViewModel(billingFormData: nil,
+                                            paymentFormStyle: DefaultPaymentFormStyle(),
+                                            billingFormStyle: DefaultBillingFormStyle())
+
+        let expectation = expectation(description: #function)
+        viewModel?.updateExpiryDateView = {
+            XCTAssertTrue(true)
```

Comment:
There is no need for an assert here, this is obviously not testing anything. If the completion is not called, expectation will time out, raising the failure that way. I'll only leave this comment, but this should apply to all the tests here